src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -138,41 +174,63 @@ export class SchoolSelectField extends React.Component {
     }
 
     return (
-      <fieldset className={fieldsetClass}>
+      <fieldset className={fieldsetClassNames}>
         <div>
-          <span>
-            {'Enter your school’s name or city to search for your school'}
-          </span>
-          <div className="search-controls">
-            <Element name="schoolSearch"/>
-            <div className="search-input">
-              <input
-                onChange={this.handleSearchInputChange}
-                onKeyDown={this.handleSearchInputKeyDown}
-                ref={input => { this.searchInput = input; }}
-                type="text"
-                value={searchInputValue}/>
-              <button
-                className="search-schools-button usa-button-primary"
-                onClick={this.handleSearchClick}>
-                {'Search Schools'}
-              </button>
-            </div>
-            <div className="clear-search">
-              <button
-                className="va-button-link start-over"
-                onClick={this.handleStartOver}>
-                {'Start Over'}
-              </button>
+          <div className={schoolSearchClassNames}>
+            <label
+              id="school-search-label"
+              className={labelClassNames}
+              htmlFor="school-search-input">
+              {'School Information'}
+              {!manualSchoolEntryChecked && <span className="schemaform-required-span">{'(*Required)'}</span>}
+            </label>
+            {showErrors && <span
```

Comment:
I tried to hack the `formConfig` to handle the error messaging but it didn't seem possible now that this field is an object. The `ui:field` is in the top level `facilityCode` but the `ui:required`  _validation_ is in `facilityCode.facilityCode`. I was able to get around this by reimplementing some of the error display logic but I'm open to adjusting this to a more idiomatic approach.

I tried adding `manualSchoolEntryChecked` to the `schema` but it tried to render a field. I couldn't figure out how to override that in the `ui:schema` so I just omitted it.

Reply:
![screen shot 2018-08-09 at 17 46 31](https://user-images.githubusercontent.com/4998130/43931549-2ec2e16e-9bfc-11e8-93db-75ecce4ba685.png)